tools/run_tests/sanity/banned_functions.py

```diff
@@ -291,7 +288,9 @@
             continue
         with open(path) as f:
             text = f.read()
-        for banned, exceptions in list(BANNED_EXCEPT.items()):
+        for banned, exceptions in list(
```

Comment:
NIT: You may want to rename the variable exceptions

Reply:
Done.